### PR TITLE
Implement OpenShift SSL passthrough tests

### PR DESCRIPTION
**Pull request by juangon**

### CODE REVIEW

File: ssl-passthrough/src/main/fabric8/service.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    expose: "true"
```
Comment (Ladicek) [inline ssl-passthrough/src/main/fabric8/service.yml]: This is copied from FMP output, right? I don't think we need/want this.

File: ssl-passthrough/src/test/java/io/thorntail/openshift/ts/ssl/passthrough/SslPassthroughIT.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+@RunWith(Arquillian.class)
+public class SslPassthroughIT {
+    @RouteURL(value = "${app.name}", path = "/hello")
+    @AwaitRoute(path="/health")
+    private URL url;
+
+    @RouteURL(value = "route-passthrough-secured", path = "/hello")
+    private URL securedRoute;
+
+    @Test
```
Comment (Ladicek) [inline ssl-passthrough/src/test/java/io/thorntail/openshift/ts/ssl/passthrough/SslPassthroughIT.java:21]: I'd expect another `@Test` that would check the unsecured route (`url`).

**Comment by Ladicek** [comment]
Generally LGTM. I think the `src/main/fabric8/*.yml` files can be slightly simplified, but that's all I have.

**Comment by juangon** [comment]
@Ladicek thanks for suggestions. Just pushed those

**Review by Ladicek — APPROVED** [review]

**Comment by Ladicek** [comment]
Thanks!

**Pull request merged**